src/Server/HTTP/HTTPServerResponse.h

```diff
@@ -108,8 +120,20 @@ class HTTPWriteBuffer : public WriteBufferFromPocoSocket
         return chunked ? 0 : fixed_length;
     }
 
+    size_t fixedLengthLeft() const
+    {
+        chassert(isFixedLength());
+
+        if (fixed_length <= count_length - offset())
+            return 0;
+        return fixed_length - count_length - offset();
+    }
+
     void setPlain(size_t buf_size = DBMS_DEFAULT_BUFFER_SIZE)
     {
+        if (count() > 0)
```

Comment:
```suggestion
        if (count() > 0 && count() != offset())
```

Or not really?

Reply:
In fact it is never called. But you are right.